Demo/EarlGreyContribs/EarlGreyContribsSwiftTests/EarlGreyContribsSwiftTests.swift

```diff
@@ -19,8 +19,8 @@ import XCTest
 
 class EarlGreyContribsSwiftTests: XCTestCase {
   override func tearDown() {
-    EarlGrey().selectElementWithMatcher(grey_anyOfMatchers([grey_text("EarlGreyContribTestApp"),
-                                                            grey_text("Back")]))
+    EarlGrey().selectElementWithMatcher(grey_anyOf([grey_text("EarlGreyContribTestApp"),
```

Comment:
Sorry I had to revert these changes owing to some test failures that were taking longer to fix, first I wanted to get the docs in place. PTAL.